version.go

```diff
@@ -33,3 +34,20 @@ func ParseVersion(ver string) (int, int, bool) {
 	}
 	return major, minor, true
 }
+
+var (
+	Version = "(devel)"
+	GitHash = "(no hash)"
+)
+
+func getVersion() (string, string) {
```

Comment:
version.go was meant to parse Fortigate version numbers, so it probably makes more sense to keep this function in the fortigate_exporter.go file.